<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1283</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Poker Game</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>741</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="CPUCards">
     <property name="spacing">
      <number>5</number>
     </property>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>741</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="PlayerCards">
     <property name="spacing">
      <number>5</number>
     </property>
    </layout>
   </widget>
   <widget class="QFrame" name="Controls">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>20</y>
      <width>351</width>
      <height>211</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="PlayButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="DeBugMode">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>170</y>
       <width>78</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Debug</string>
     </property>
    </widget>
    <widget class="QLabel" name="roundCount_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>301</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>72</pointsize>
      </font>
     </property>
     <property name="text">
      <string>POKER</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="GameControls">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>270</y>
      <width>351</width>
      <height>211</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="next">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Play Round</string>
     </property>
    </widget>
    <widget class="QPushButton" name="swapCard">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Swap</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sortByGroup">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Sort by Group</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sortByValue">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Sort by Value</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Restart">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>130</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>END</string>
     </property>
    </widget>
    <widget class="QLabel" name="swapsLeft">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SWAPS</string>
     </property>
    </widget>
    <widget class="QLabel" name="round">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ROUND</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="Results">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>250</y>
      <width>351</width>
      <height>261</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="roundCount_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="text">
      <string>FINAL SCORE</string>
     </property>
    </widget>
    <widget class="QLabel" name="winner">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>FINAL SCORE</string>
     </property>
    </widget>
    <widget class="QLabel" name="playerScore">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>FINAL SCORE</string>
     </property>
    </widget>
    <widget class="QLabel" name="cpuScore">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>FINAL SCORE</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="display">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>351</width>
      <height>211</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <widget class="QPushButton" name="skip">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>160</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QLabel" name="roundCount">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="cpuScore_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>130</y>
       <width>101</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CPU SCORE</string>
     </property>
    </widget>
    <widget class="QLabel" name="playerScore_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>121</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PLAYER SCORE</string>
     </property>
    </widget>
    <widget class="QLabel" name="roundInfo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>301</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CPU SCORE</string>
     </property>
    </widget>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget_2</zorder>
   <zorder>Controls</zorder>
   <zorder>Results</zorder>
   <zorder>GameControls</zorder>
   <zorder>display</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
